src/segment.go

```diff
@@ -130,6 +130,8 @@ const (
 	Angular SegmentType = "angular"
 	// PHP writes which php version is currently active
 	PHP SegmentType = "php"
+	// RegQuery queries the Windows registry.
+	RegQuery SegmentType = "regquery"
```

Comment:
I would rename this to `winreg`, it's about the registry, the fact that we query it is an implementation detail.